src/rest-server/src/models/kubernetes.js

```diff
@@ -5,8 +5,25 @@ const axios = require('axios');
 const {Agent} = require('https');
 const {URL} = require('url');
 const {apiserver} = require('@pai/config/kubernetes');
+const status = require('statuses');
+const createError = require('@pai/utils/error');
 const logger = require('@pai/config/logger');
 
+const patchOption = {
+  headers: {
+    'Content-Type': 'application/merge-patch+json',
+  },
+};
+
+const rethrowResponseError = (error) => {
```

Comment:
I think the error handler could be enabled to each k8s requests (could be added to `getClient` function).

Reply:
As offline discuss, will leave GC to watch dog

Reply:
Ok, I was talking about another problem about axios.
Axios will throw errors with wrong stack trace and error message only conclude status code. And only this two field will be logged to console if the error is not handled.
I think we need to modify the error message and add some request info into the message (at least the request url). Another options is to log the failed request info into console and keep the original error object.

By the way, I think it is not a good idea to throw a express http error here.
The model file may be used by other models and utilities which may need the orignial axios error object with all information.

Reply:
OK, I have changed the error handling logic. Modify the error message and add some request info into the message. Don't throw express http error here